lib/heimdall_tools/help/aws_config_mapper.md

```diff
@@ -0,0 +1,20 @@
+  aws_config_mapper pulls Ruby AWS SDK data to translate AWS Config Rule results into HDF format json to be viewable in Heimdall
+
+Setting up AWS Config rules for the aws_config_mapper:
+ 
+  The mapper looks for specific tags on AWS Config Rule resources to pass on to the HDF format:
+    - `severity`:   may be set to `Info`, `Low`, `Medium`, `High`, or `Critical`
+    - `cce_id`:     is passed through to HDF tags as-is
+    - `check`:      is passed through to HDF tags as-is
+    - `cis_impact`: is passed through to HDF tags as-is
+    - `cis_rid`:    is passed through to HDF tags as-is
+    - `fix`:        is passed through to HDF tags as-is
+    - `rationale`:  is passed through to HDF tags as-is 
+    - `cweid`:      is passed through to HDF tags as-is
+    - `confidence`: is passed through to HDF tags as-is
+    - `cis_level`:  is parsed as an integer and passed to HDF tags
+    - `nist`:       expects the format `AC-04:SC-07:SC-07-03` to translate to `AC-04, SC-07, SC-07(03)` before passing to HDF tags
+  
+Examples:
+
+  heimdall_tools aws_config_mapper -o aws_config_results.json
```

Comment:
Some information on how to actually configure the aws config mapper authentication (I assume it has to authenticate to the service it is scanning) would be useful here.

Reply:
Please see updates to this file

Reply:
👍